containers/ansible.cfg

```diff
@@ -1,5 +1,5 @@
 [defaults]
-vault_password_file = .vault_pass
+#vault_password_file = .vault_pass
```

Comment:
Is this no longer needed?

Reply:
I need to re-examine where an dhow it's needed. For the moment I commented it out since it was making life more difficult.